scripts/build/electron-installer-debian-linux.sh

```diff
@@ -62,10 +62,15 @@ then
   usage
 fi
 
+if [ ! -d "$PWD/node_modules" ]; then
```

Comment:
Good catch! I actually made the `check-dependency.sh` script check for the dependency in `./node_modules/.bin` and removed this check, since if the check dependency script succeeded, then there is no way this would fail.